README.md

```diff
@@ -14,5 +14,23 @@ Developed by:
 * [simon holmes à court (@simonahac) | Twitter](https://twitter.com/simonahac)
 * [Steven Tan (@chienleng) | Twitter](https://twitter.com/chienleng)
 
+## Development
```

Comment:
Add a top and bottom horizontal rule for the Development section. To make it more readable.